src/main/resources/userEmailTemplate-feedbackSessionUnpublished.html

```diff
@@ -0,0 +1,16 @@
+Hello ${userName},
+
+<p/>
+${instructorFragment}
+
+<p/>
+The feedback responses for the following feedback session is now closed for viewing.
+<br>&nbsp;&nbsp; Course: [${courseId}]${courseName}
+<br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
+<br>
+<p/>
+If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
+<p/>
+Regards,
+<br>
+TEAMMATES Team.
```

Comment:
@tarun14110, I believe the only way for a session to get unpublished is through `InstructorFeedbackUnpublishAction`